src/patternfly/components/Select/select-toggle-typeahead.hbs

```diff
@@ -1,13 +1,15 @@
-{{#if select--ItemIsSelected}}
-	{{#> form-control controlType="input"
-		input="true"
-		form-control--modifier="pf-c-select__toggle-typeahead"
-		form-control--attribute=(concat 'type="text" id="' id '-typeahead" aria-label="Type to filter" value="' select-typeahead--Placeholder '"')}}
-	{{/form-control}}
-{{else}}
-	{{#> form-control controlType="input"
-		input="true"
-		form-control--modifier="pf-c-select__toggle-typeahead"
-		form-control--attribute=(concat 'type="text" id="' id '-typeahead" aria-label="Type to filter" placeholder="' select-typeahead--Placeholder '"')}}
-	{{/form-control}}
-{{/if}}
+<form class="pf-c-select__toggle-form">
```

Comment:
Can you check the indentation in this file? Looks like tabs are creating too many spaces.